src/server.ts

```diff
@@ -1,16 +1,24 @@
 import express, { Request, Response } from 'express';
 import dotenv from 'dotenv';
 import routes from './routes/index'
-
+import { AdminsSeeders } from './database/seeders/Admins.seeds';
+import db from './database/models/index';
+const addAdmins = ()=>{
+  AdminsSeeders.forEach(admin =>{
+    db.Admins.create(admin)
+  })
+}
 
 dotenv.config();
 
 const app = express();
 const port = process.env.PORT || 7000;
-
+const databaseUrl= process.env.DATABASE_URL
```

Comment:
How are you having this `DATABASE_URL`, yet the your current configurations didn't have that environment?